helmnet/dataloaders.py

```diff
@@ -27,10 +28,21 @@ class EllipsesDataset(Dataset):
     """Dataset of oversimplified skulls."""
 
     def __init__(self):
-        self.all_sos = []
+        self._all_sos = []
+        self.all_sos_numpy = []
```

Comment:
@sio13 I remember that I wasn't very happy with this thing of having a copy of the sos maps both as numpy.array and torch.tensor, is there a fundamental reason for doing so that I'm forgetting?

If it is just for explicitly moving them to cuda, it feels that the proper place for doing so is in the collate function of the dataloader rather than here

Reply:
@astanziola Please, correct me if I am wrong, but I think there was a copy of both NumPy and torch tensors. When we want to iterate over Dataset, we have to call a initialize `self.all_sos`:
```
self.all_sos = torch.from_numpy(self.all_sos_numpy).unsqueeze(1).float()
```
That indicates the instance of `EllipsesDataset` holds both types of the data tensor. 

I placed the line `self.all_sos_numpy = []` just for consistency with `all_sos`, it shouldn't add more data into the RAM.

Reply:
Ok it makes sense. Still a bit weird though, but it feels like that's the topic for another PR.